testapi.pm

```diff
@@ -1524,18 +1524,17 @@ sub power {
     query_isotovideo('backend_power', {action => $action});
 }
 
-=head2 assert_shutdown
+=head2 check_shutdown
 
-  assert_shutdown([$timeout]);
+  check_shutdown([$timeout]);
 
 Periodically check backend for status until C<'shutdown'>. Does I<not> initiate shutdown.
-Default timeout is 60s
 
-Returns C<undef> on success, throws exception on timeout.
+Returns true on success and false if C<$timeout> timeout is hit. Default timeout is 60s.
 
 =cut
 
-sub assert_shutdown {
+sub check_shutdown {
```

Comment:
You will need to use Test::MockModule, mock the query_isotovideo method from autotest to return the expected values. An example on how to do that can be seen in the [testapi tests](https://github.com/os-autoinst/os-autoinst/blob/master/t/03-testapi.t#L157)